Purpose of these dependencies is to make sure that the linker
is run serially over these binaries instead of in parallel.
OBS workers run out of memory as each executable seems to require
upward of 5G RAM

Index: node-v18.14.0/node.gyp
===================================================================
--- node-v18.14.0.orig/node.gyp
+++ node-v18.14.0/node.gyp
@@ -1166,6 +1166,7 @@
         'node_dtrace_header',
         'node_dtrace_ustack',
         'node_dtrace_provider',
+        'cctest',
       ],
       'includes': [
         'node.gypi'
@@ -1312,6 +1313,7 @@
         'node_dtrace_header',
         'node_dtrace_ustack',
         'node_dtrace_provider',
+        'fuzz_env',
       ],
 
       'includes': [
@@ -1381,6 +1383,7 @@
         '<(node_lib_target_name)',
         'deps/histogram/histogram.gyp:histogram',
         'deps/uvwasi/uvwasi.gyp:uvwasi',
+        'embedtest'
       ],
 
       'includes': [
